.github/workflows/playwright.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           tools: composer
       - uses: subosito/flutter-action@v2
         with:
-          flutter-version: "2.8.1"
+          flutter-version: "3.0.5"
```

Comment:
https://github.com/subosito/flutter-action/issues/179#issuecomment-1195120616

Reply:
alternative : if newer version of flutter is needed install dart before from an official GHA https://github.com/dart-lang/setup-dart